### DISCUSSION TITLE
Yamaha Reface Dx is not listed in Setup 1.15.0

### DISCUSSION THREAD

**Discussion by SynthSpace** [christofmuc/KnobKraft-orm#158] [General]
Dear Christof, I wanted to let you know that.
In 1.15.0 the list ends at Yamaha DX7II... The Reface Dx is missing.

I rolled back too the previous version there is present and works :) 
Thanks you for the awesome software!

**Comment by christofmuc** [comment]
Oh, thanks for pointing this out! Must have been a loss when I tested the adaptation version of the reface DX - but I didn't retire it yet, so that's a bug. I'll make a 1.15.1 with that fixed!

**Comment by SynthSpace** [comment]
My pleasure, thank you for making life a lot easier with my Dx!! Ill keep an eye out for the update 😁👏👏 Blessings.

**Comment by christofmuc** [comment]
Closing this as fixed.
